extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -199,6 +199,14 @@ export class SafeframeHostApi {
     this.registerSafeframeHost();
   }
 
+  /**
+   * Returns true if the given window matches the Safeframe's content window.
+   * @return {Boolean}
+   */  
+   equalsSafeframeContentWindow(otherWindow) {
+    return otherWindow === this.iframe_.contentWindow;
```

Comment:
I think there's a lint error here.

Reply:
Yup, and some other tests are still failing. Working on updating them. Thanks for looking :)